src/OrbitGl/TrackManagerTest.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TRACK_MANAGER_TEST_H_
+#define ORBIT_GL_TRACK_MANAGER_TEST_H_
+
+#include <cstdint>
+#include <memory>
+
+#include "ClientModel/CaptureData.h"
+
+namespace orbit_gl {
+
+struct TrackTestData {
+  static constexpr uint64_t kCallstackId = 1;
+  static constexpr uint64_t kFunctionAbsoluteAddress = 0x30;
+  static constexpr uint64_t kInstructionAbsoluteAddress = 0x31;
+  static constexpr int32_t kThreadId = 42;
+  static constexpr const char* kFunctionName = "example function";
+  static constexpr const char* kModuleName = "example module";
+  static constexpr const char* kThreadName = "example thread";
+
+  static constexpr size_t kTimerOnlyThreadId = 128;
+  static constexpr const char* kTimerOnlyThreadName = "timer only thread";
```

Comment:
I don't really like that these have been moved to a header. These are just random values used in some test, where the implementer needed that specific set of values. I don't see a reason to share them between tests, and indeed they are only used by `TrackManagerTest`.

Reply:
Ah you need them to expose the values used in `GenerateTestCaptureData` I guess?

Reply:
Yes. They are now in `TrackTestData.h`, mainly because I want to share those between tests without copy / pasting everything... it's currently quite specific to two tests, but I assume we can extend them later to contain enough data to be used in other tests around tracks as well.